app/src/ui/multi-commit-operation/multi-commit-operation-manager.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import { assertNever } from '../../lib/fatal-error'
+import { MultiCommitOperationKind } from '../../models/multi-commit-operation'
+import { Squash } from './squash'
+import { IMultiCommitOperationProps } from './multi-commit-operation'
+
+/** A component for initiating and performing a rebase of the current branch. */
+export class MultiCommitOperationManager extends React.Component<
```

Comment:
Using the suffix `Manager` for a view/component feels odd… what about `MultiCommitOperationView` or `MultiCommitOperationWrapper` since it wraps subclasses of `MultiCommitOperation` 🤔 

Another option… this is a `MultiCommitOperation` and the existing `MultiCommitOperation` becomes `BaseMultiCommitOperation`?

Reply:
On another project I worked on, we had the same flow for something similar and called it `Manager` (and another team called it `Process`). 😛  Thus, this feels natural to me. But.. I do like `BaseMultiCommitOperation`. It is clearer what the intent of it is. Refactored to  the second option. 😄